app/assets/stylesheets/index.scss

```diff
@@ -68,10 +69,37 @@ a {
   width: 50%;
 }
 
-.search-form {
-  display: inline;
-  float: right;
-  padding-left: 5px;
-  padding-right: 5px;
-  padding-top: 5px;
+// #search-form {
+//   display: block;
+//   float: right;
+//   padding-left: 5px;
+//   padding-right: 5px;
+//   padding-top: 5px;
+// }
+
+.search_field {
+  background-color: $black;
+  display: block;
+}
+
+.search_button {
+  background-color: $black;
+
+  display: block;
+  margin-bottom: 0;
+  background-color: $black;
+}
+
+#keyword_search_button {
+  background-color: $green;
+  border-radius: 5px;
```

Comment:
Properties should be ordered background-color, border, border-radius, color, display, padding-bottom, padding-top